app/src/components/fineTunes/FineTunesTable.tsx

```diff
@@ -86,13 +86,31 @@ const FineTunesTable = ({}) => {
 
 export default FineTunesTable;
 
-export const getStatusColor = (status: FineTuneStatus) => {
-  switch (status) {
-    case "DEPLOYED":
-      return "green.500";
-    case "ERROR":
-      return "red.500";
-    default:
-      return "yellow.500";
-  }
+export const FTStatus = (props: { status: FineTuneStatus }) => {
+  const getStatusColor = (status: FineTuneStatus) => {
+    switch (status) {
+      case "DEPLOYED":
+        return "green.500";
+      case "ERROR":
+        return "red.500";
+      default:
+        return "yellow.500";
+    }
+  };
+
+  const statusText = (status: FineTuneStatus) => {
```

Comment:
Defining these functions outside the FTStatus component will speed up rendering, because we won't have to redefine this function every time we rerender. This isn't a big deal, because we don't render very many of these at once.

Reply:
Yeah not sure why I moved this inside. I think Cursor did it for me but there's no reason for it. Will fix.